bigtable/google/cloud/bigtable/row_data.py

```diff
@@ -274,6 +274,10 @@ def consume_next(self):
 
             self._validate_chunk(chunk)
 
+            if ("ReadRowsIterator" in
```

Comment:
This is kind of a bizarre check. If an unrelated library defined a different class with the same name, it would pass. Likewise, subclasses of `ReadRowsIterator` that do not include that exact string in the class name would fail.
Please either check for the presence of the method, or check the actual class.

Option 1:

```python
if hasattr(self._response_iterator, 'set_start_key'):
```

Option 2:

```python
if isinstance(self._response_iterator, ReadRowsIterator):
```

If there is some reason that you really need the check in this specific form that is not covered by either of the two options above, then please elaborate. :-)